verilog/CST/type.h

```diff
@@ -108,6 +139,18 @@ const verible::SyntaxTreeNode* GetParamListFromInstantiationType(
 const verible::SyntaxTreeLeaf& GetSymbolIdentifierFromEnumName(
     const verible::Symbol& enum_name);
 
+// Returns symbol identifier node for the type name from node tagged with
+// kInstantiationType (if exists) or return nullptr.
+//- e.g some_type x; return "some_type".
```

Comment:
General readability suggestion for these examples: double quote both the enclosing text and the returned text, so:
`// e.g. from "this snippet..." return "this piece"`.